.github/workflows/sdk_generation.yaml

```diff
@@ -22,4 +22,5 @@ jobs:
       speakeasy_version: latest
     secrets:
       github_access_token: ${{ secrets.GITHUB_TOKEN }}
+      nuget_api_key: ${{ secrets.NUGET_API_KEY }}
```

Comment:
is it needed here? 🤔

Reply:
That was plumbed in by the `speakeasy configure publishing` CLI command, so I have no clue! :sweat_smile: